Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    public float Speed = 3f;
+    public float AngularSpeed = 360f;
+    void Update()
+    {
+        // movement direction
+        var movement = new Vector3(Input.GetAxis("Horizontal"), 0, Input.GetAxis("Vertical"));
```

Comment:
the diagonal movement is faster than when moving along one of the axes
if movement vector's length is above 1, you should normalize it

Reply:
-0.5 points